ZSSRforKernelGAN/ZSSR.py

```diff
@@ -370,3 +315,88 @@ def base_change(self):
                 self.base_ind += 1
 
             print('base changed to %.2f' % self.base_sf)
+
+    def epoch_Z(self, kernels=None):
+        # Increment epcho number of ZSSR
+        self.epcho_num_Z += 1
```

Comment:
why not to use self.iter that is configured in the loop?

Reply:
in order to allow different schemes of training where ZSSR epcho might not be called any round or more then once in one round and so on...
the self.iter in that is configured in the look is from previous code and I plan to remove it